dev-resources/template/1_db.yml

```diff
@@ -1,12 +1,11 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: "Postgres DB for SQL LRS"
 Parameters:
-  # Networking
   VPCId:
     Description: "VPC on which to run SQL LRS"
     Type: AWS::EC2::VPC::Id
   DBSubnets:
-    Description: Subnets on which to run services
+    Description: Subnets on which the DB is running
```

Comment:
This wasn't correct before, but the updated copy isn't really accurate. Should be something like: "Subnets on which to run the DB initialization Lambda function"

Reply:
addressed via https://github.com/yetanalytics/lrsql/pull/276/commits/84a3fd8e8aaf995eb84faa9df0e158cbe4f9113e